kitsune/community/jinja2/community/email/first_answer.html

```diff
@@ -10,8 +10,8 @@
 
   <p>
     {% trans %}
-      Mozilla support is fully powered by our volunteer community that is
-      helping hundreds of users everyday. This community is formed by
+      Mozilla support is fully powered by our volunteer community that helps
+      hundreds of users everyday. This community is formed by
```

Comment:
'everyday' should be 'every day'. Everyday is an adjective that means "common" or "usual", but  'every day' is an adverbial phrase that means "each day".